Tacta.EventStore/Domain/DomainEvent.cs

```diff
@@ -9,27 +9,18 @@ public abstract class DomainEvent : IDomainEvent
 
         public string AggregateId { get; }
 
-        [JsonIgnore] public int Version { get; private set; }
-
-        [JsonIgnore] public long Sequence { get; private set; }
-
         public DateTime CreatedAt { get; set; }
 
        
         protected DomainEvent(string aggregateId)
         {
-            (Id, AggregateId, CreatedAt, Version, Sequence) = (Guid.NewGuid(), aggregateId, DateTime.Now, 0, 0);
+            (Id, AggregateId, CreatedAt) = (Guid.NewGuid(), aggregateId, DateTime.Now);
         }
 
         [JsonConstructor]
         protected DomainEvent(Guid id, string aggregateId, DateTime createdAt)
         {
             (Id, AggregateId, CreatedAt) = (id, aggregateId, createdAt);
         }
-
-        public void WithVersionAndSequence(int version, long sequence)
```

Comment:
is this a breaking change?

Reply:
Yes, this is a breaking change in terms that this is not needed anymore. Since there is a pull request with another breaking change (https://github.com/tacta-io/Tacta.EventStore/pull/25), can we combine these two PR's together, so as to not make a two major versions one after another?